arches/app/etl_modules/base_import_module.py

```diff
@@ -251,6 +254,16 @@ def read(self, request):
             result["summary"]["files"][content.name] = {"size": (self.filesize_format(content.size))}
             result["summary"]["cumulative_excel_files_size"] = self.cumulative_excel_files_size
             default_storage.save(os.path.join(self.temp_dir, content.name), File(content))
+
+        for file in result["summary"]["files"]:
+            if file.split(".")[-1] == "xlsx":
+                uploaded_file_path = os.path.join(self.temp_dir, file)
+                graphid = self.validate_uploaded_file(uploaded_file_path)
+                if graphid is None:
+                    title = _("Invalid Uploaded File")
+                    message = _("The key information is missing. Make sure the uploaded excel template is in the right format.")
```

Comment:
I think there are cases when the information is maybe not missing, but just incorrect. Maybe this should be something like: `This file has missing information or invalid formatting. Make sure the file is complete and in the expected format`

Reply:
updated